apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html

```diff
@@ -25,6 +25,14 @@ <h2>Header</h2>
                 <div *ngIf="fieldsAndVisibility.header.gstin" class="form-group">
                     <input type="checkbox" [(ngModel)]="customTemplate.sections['header'].data['gstin'].display" (ngModelChange)="onChangeFieldVisibility(null, null, null)" checked /> GSTIN
                 </div>
+                <ng-template>
+                    <div class="form-group clearfix customField">
```

Comment:
use hypen-case class

Reply:
customField this class too many places we we cant change right now